pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/http/AsyncHttpConnector.java

```diff
@@ -114,58 +107,14 @@ public boolean keepAlive(InetSocketAddress remoteAddress, Request ahcRequest,
         });
 
         serviceNameResolver = new PulsarServiceNameResolver();
-        if (conf != null && StringUtils.isNotBlank(conf.getServiceUrl())) {
+        if (StringUtils.isNotBlank(conf.getServiceUrl())) {
```

Comment:
It seems that the` ClientConfigurationData conf` may be null in `PulsarAdmin`. Shall you keep the judgement to avoid NPE ?

Reply:
It was always non-null.

Reply:
@HQebupt